packages/fxa-content-server/app/scripts/views/mixins/third-party-auth-mixin.js

```diff
@@ -148,7 +150,7 @@ export default {
       access_type: 'offline',
       prompt: 'consent',
       response_type: 'code id_token',
-      response_mode: 'form_post',
+      response_mode: 'form_post'
```

Comment:
your auto formatting might be different than my auto formatting for this trailing comma 😄